imports/plugins/core/ui/client/components/checkbox/checkbox.js

```diff
@@ -20,13 +30,15 @@ class Checkbox extends Component {
 
   render() {
     return (
-      <label>
+      <label htmlFor={this.id}>
```

Comment:
I was under the impression that you do not need `for` if the label wraps the input 🤷‍♂️

Reply:
I was surprised by it too, but the rules state otherwise.